README.md

```diff
@@ -101,23 +120,30 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
+[x] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
-[] `document.getElementsByClassName("post")[0].innerHTML`
+[x] `$(".post").html()`
+[x] `document.getElementsByClassName("post")[0].innerHTML`
 [] `document.getElementsByClassName("post").innerHTML`
 ```
 
 ### Question #9
 
-Using jQuery, add an event listener for clicks on the button with the id
+**Using jQuery, add an event listener for clicks on the button with the id
 'greeting'. When the event happens, the code should append a paragraph to the
-body, that says "hello".
+body, that says "hello".**
 
 Your Answer:
 ```js
-// your code here
+write code here
+
+<button id=greeting>push me</button>
+
+$('#greeting').on('click', function(){
+  $('<p>hello</p>').appendTo($('body'))
```

Comment:
:+1:  
I had to lookup `appendTo`.  I usually use `$('body').append($('<p>hello</p>');`.  Either works.